### ISSUE TITLE
Fetch list of org repos for homepage

### ISSUE THREAD

**Issue by bennypowers** [RedHat-Israel/RedHat-Israel.github.io#37]
the list of repos on the homepage should automatically include everything under redhat-israel org, perhaps except for this repo itself. we can just fetch the list of repo names and add them to the list

**Comment by TomerFi** [comment]
suggestion: a list of relevant redhat organizations (like this one).

the repository list can be concatenated from all listed organizations,
filtering out private repos, archived repos, and perhaps a manual maintained list of exclusions.

this list can be merged with the current manual maintained list we have.

**Comment by bennypowers** [comment]
i like the idea. how do we filter for redhat-israel contributors?

**Comment by TomerFi** [comment]
how about the members here?

when creating the data for the site,
we'll retrieve all the public non-archived repos from a manual maintained list of organizations.
filtering out repos from a manual maintained excluded repos list,
and grabbing only repos including contributions from our members list.

it might take some io from gh api, but I think it's doable.
maybe we should a pin on this until we tackle the token limitations issue we discussed earlier this week.

**Comment by TomerFi** [comment]
re-thinking this...
if someone drops from this organization,
unless we have another member contributing in the same repo,
it won't show up anymore from the next build.

**Comment by bennypowers** [comment]
js Set has unique entries, so just make a Set of the manual entries and add each result from your heuristic, you don't have to worry about duplicates then

**Comment by TomerFi** [comment]
of course, but the data is not persisted, if we re-generate the entries set for every build based on members in this org,
this means this if I leave this org, in the next build, unless we have another member contributing in the same exact repos, they will not appear in the new set.

**Comment by TomerFi** [comment]
maybe we should discuss persisting our data.
our current data fetching is a lengthy one, and we're going to add on top of it.

so... maybe it's time to consider persisting this data somewhere and updating it nightly or some other schedule, having this site build quicker by simply retrieving the data from the persistence target.

we can do it with another repo that will host the persistent data and some workflows updating it.
we can also do it in this repo, but I think it will be messy.
